openapi.yaml

```
@@ -50,6 +55,58 @@ tags:
       description: Find out more about Juju users.
       url: https://juju.is/docs/olm/manage-users
 paths:
+  /deploy:
+    post:
+      tags:
+        - application
+      description: Deploy an application. If a local resource is provided, details
+        required for uploading the validated resource will be returned.
+      # This needs to return info about local resources to be uploaded by the client
+      # post-deploy (PendingResourceUpload)
```

```suggestion
      description: |
        Deploy an application. 

        Returns a simple response with a location header pointing to the deployed
        application.

        If the deployment relies upon local resources being uploaded, clients should
        retrieve the URL in the location header send in the response in order to
        learn the URL resources should be POSTed too after deploy (see `uploadResource`
        operation).
```

Note the `description: |` part here too -- can you please ensure that's tidied up anywhere you've got a newline in a description?